nipype/interfaces/niftyreg/base.py

```diff
@@ -47,8 +47,9 @@ def no_nifty_package(cmd='reg_f3d'):
 class NiftyRegCommandInputSpec(CommandLineInputSpec):
     """Input Spec for niftyreg interfaces."""
     # Set the number of omp thread to use
-    omp_core_val = traits.Int(desc='Number of openmp thread to use',
-                              argstr='-omp %i')
+    omp_core_val = traits.Int(int(os.environ.get('OMP_NUM_THREADS', '1')),
```

Comment:
I personally don't like this because it makes nipype's behavior more dependent on hidden state, which means that different people testing and using the code may see different behavior.

But I'll leave the final call to @satra.